data/release-notes/enterprise-server/3-2/0.yml

```
@@ -59,7 +59,7 @@ sections:
         - |
           Git Credential Manager (GCM) Core versions 2.0.452 and later now provide security-hardened credential storage and multi-factor authentication support for {% data variables.product.product_name %}.
```

Same as above, let's drop the reference here too.
```suggestion
          Git Credential Manager (GCM) versions 2.0.452 and later now provide security-hardened credential storage and multi-factor authentication support for {% data variables.product.product_name %}.
```